src/remote.go

```diff
@@ -142,11 +142,7 @@ func (r *Remote) changes(startChangeId int64) (chan *drive.Change, error) {
 func buildExpression(parentId string, typeMask int, inTrash bool) string {
 	var exprBuilder []string
 
-	if inTrash || (typeMask&InTrash) != 0 {
```

Comment:
Yes, `list` and `list --trashed` now work in both root and nested directories. Please test this yourself.

What broke `list --trashed` was two separate small bugs. First, `list --trashed` wasn't working in nested directories because it was looking for the directory in trash rather than just its containing files in trash. So I switched `list` to use its default path resolver `FindByPath` instead of `FindByPathTrashed`.

Second, `list --trashed` returned all trashed items in all directories. I found in `buildExpression` that a parent directory was not specified if searching `inTrash`. Adding it back to match the regular case fixed the issue. I also removed the `(typeMask&InTrash) != 0` check because I felt it shouldn't contradict the `inTrash` parameter and simplified the expression even more.